Dev qz

imppresser opened the pull request:
See the Commits.

imppresser committed:
First View.

imppresser committed:
Questions about the project.

imppresser committed:
增加进入 登录 和 注册 界面的两个入口，对登录界面的edittext控件进行了初步的修改。

imppresser committed:
Replace "signup" as "register".

imppresser committed:
fix the ids collision from "HabitItemView.java".

imppresser committed:
EZ UI setup for the login screen.

imppresser committed:
Add hidePassword icon to the login screen.

imppresser committed:
UI fix & EZ settings for RegisterHomeScreen.

imppresser committed:
UI :
Completed the RegisterHomeScreen UI design.
Completed the RegisterPasswordScreen UI design.
Fix the dimen.xml, ids.xml. strings.xml.
Controller :
Add several ActionID in AccountController.

imppresser committed:
UI :
EZ UI settings for RegisterUserInfoScreen.

imppresser committed:
UI :
Done UI settings for RegisterUserInfoScreen.
Done UI settings for RegisterOKScreen.

Waiting for UI additional content for register & login part.

CMCCHEMC commented on .idea/.name:
```diff
@@ -1 +1 @@
-PersistNoteWorkspace
\ No newline at end of file
+PersistNote
```
idea目录下的文件各人配置不同不要提交

CMCCHEMC commented on app/src/main/java/com/dv/persistnote/business/LoginScreen.java:
```diff
@@ -0,0 +1,299 @@
+
+    private RelativeLayout mContainer;
+
+    private RelativeLayout containerPhoneNumber;
```
类的内部成员变量统一用 mXX，静态变量用 sXX 这样的命名

CMCCHEMC commented on app/src/main/java/com/dv/persistnote/business/RegisterHomeScreen.java:
```diff
@@ -0,0 +1,311 @@
+import java.lang.reflect.Field;
+
+/**
+ * Created by Admin2 on 2016/3/29.
```
文件名也统一写上自己名字啦，可以在AS里面配置的

CMCCHEMC commented on app/src/main/res/values/dimen.xml:
```diff
@@ -9,19 +9,70 @@
+    <dimen name="h4">12dp</dimen>
+    <dimen name="h5">10dp</dimen>
 
-    <dimen name="common_margin_16">16dp</dimen>
```
这个还是留着16吧，后面会有更多的通用margin的

CMCCHEMC commented on questions.txt:
```diff
@@ -0,0 +1,2 @@
+1.DependPolicy?
```
haha 这个也不要提上来

CMCCHEMC commented on app/src/main/java/com/dv/persistnote/business/RegisterHomeScreen.java:
```diff
@@ -0,0 +1,311 @@
+                }
+            }
+        });
+        try {
```
封装成一个工具类方法在公共的工具类里，其他地方应该也会需要用到的

CMCCHEMC commented on app/src/main/java/com/dv/persistnote/business/RegisterUserInfoScreen.java:
```diff
@@ -0,0 +1,235 @@
+        lpC3.addRule(RelativeLayout.BELOW, R.id.register_u_rl_user_sex);
+        lpC3.addRule(RelativeLayout.CENTER_HORIZONTAL);
+
+        okButton = new CircleView(getContext(), ResTools.getDimenInt(R.dimen.common_cv_radius),
```
这里其实只是需要要给圆形的有颜色的背景，通过xml配置shape或者是定义一个圆形的drawable都可以满足，这样只需要一个ImageVIew，src是箭头的图片，背景是这个圆形的drawable就好了。布局这边也可以少一个view和少一个VIewGroup，条理会简单很多

CMCCHEMC commented on .idea/vcs.xml:
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="" />
```
同上，所有配置的都不提交

CMCCHEMC commented:
加了一些review意见，你看看

imppresser commented:
看到了

imppresser committed:
UI:
Little work for Delete & Hide function,
assuming EZ Touch container can help users although the viewgroup gonna be rough.

imppresser committed:
UI:
完成RegisterHomeScreen和LoginScreen的新规范设置。

imppresser committed:
UI:
完成RegisterPasswordScreen的新规范设置。

imppresser committed:
UI:
完成登录&注册

imppresser committed:
UI:
增加忘记密码入口
